src/OneExplorer/ConfigObject.ts

```diff
@@ -325,6 +325,24 @@ export class ConfigObj {
       }),
     });
 
+    // NOTE
+    // Shows <model>.trace.json
+    // REQUIRES: <model>.tvn be written in the config file.
```

Comment:
@jyoungyun The tvn file should be written in the config file to show it. Will it be enough?

Reply:
Sure. I'm considering profiling with the tvn file created in ONE-VSCode, so it is satisfied with your constraints.